### ISSUE TITLE
Possible incompatibility with StyledPlayerList

### ISSUE THREAD

**Issue by Kaden-Skinner** [DrexHD/Vanish#17]
Vanish does not remove the player from the tablist.

**Comment by Kaden-Skinner** [comment]
Is this permission on by default?

**Comment by DrexHD** [comment]
No, vanish uses [fabric-permissions-api](https://github.com/lucko/fabric-permissions-api). If you don't have any mods like [luckperms](https://modrinth.com/mod/luckperms) or [player roles](https://modrinth.com/mod/player-roles) or any permission mod that uses it, only OP players can see vanished players. You could join my [discord](https://discord.gg/HeZayd6SxF) if you need support for this mod.

**Comment by Kaden-Skinner** [comment]
![image](https://github.com/DrexHD/Vanish/assets/78925970/119b6521-0686-46d7-a915-77dbb12dc70d)
So, it was a permission problem. However, it does show that there are 2 players online instead of 1.

**Comment by DrexHD** [comment]
Replace `%server:online%` placeholder with `%vanish:online%`.

**Comment by Kaden-Skinner** [comment]
Works. Thank you!
